packages/espressocash_backend/share_escrow/pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: share_escrow
```

Comment:
Why does it need to be a separate package?

Reply:
You're right, I though it was a good idea to separate the ix in another package. I've updated and moved share ix inside the main lib